libfuse/mounter.go

```diff
@@ -87,7 +87,8 @@ func (m *mounter) Unmount() (err error) {
 			_, err = exec.Command(
 				"/usr/sbin/diskutil", "unmountDisk", "force", dir).Output()
 		case "linux":
-			_, err = exec.Command("fusermount", "-ul", dir).Output()
+			// Lazy unmount; will unmount when KBFS is no longer in use
+			_, err = exec.Command("fusermount", "-u", "-z", dir).Output()
```

Comment:
@taruti is this ok? my fusermount(1) shows the option is `-z`, which is pretty confusing since `umount` uses `-l`. hopefully it isn't os-dependent

Reply:
It'd be nice to use the [syscall unmount](https://github.com/keybase/fuse/commit/e3996deee1b03bec702b6a4d4f5b6f2154895e41#diff-eb35e1f2018531b3563fe5d935fa14d6) like [here](https://github.com/keybase/kbfs/blob/c2afd3aee19dbc113edd2b148db63834c0d08ef1/redirector/main.go#L287), but I guess can be in a separate PR.